src/main/webapp/src/components/lyrics.js

```diff
@@ -1,15 +1,77 @@
 import React from "react";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/styles";
+import { lighten } from '@material-ui/core/styles';
 import Typography from "@material-ui/core/Typography";
+import LinearProgress from '@material-ui/core/LinearProgress';
+import AudioPlayer from 'material-ui-audio-player';
+import axios from "axios";
+
+const styles = () => ({
+  root: {
+    minWidth: "300px",
+    display: "flex",
+    flexDirection: "column"
+  },
+  audio: {
+    margin: "20px 0 20px 0",
+  }
+});
+
+const StyledLinearProgress = withStyles({
+  root: {
+    backgroundColor: lighten('#000000', 0.5), 
+  },
+  bar: {
+    background: '#000000',
+  },
+})(LinearProgress);
```

Comment:
I feel it's a little bit more organized to keep the styles in a CSS file and import them here. So below you'd have something like `<LinearProgress className="StyledLinearProgress" />`.

However, from what I've found online, it's a matter of personal preference, so feel free to keep using inline styles.

Reply:
I have an issue #30, so all such things will be moved to the separate files

Reply:
SGTM